stubs/tests/ResourceTest.php

```diff
@@ -57,6 +57,29 @@ public function test_users_can_edit_resources_belonging_to_them()
         $response->assertRedirect(localized_route('resources.show', $resource));
     }
 
+    public function test_users_can_translate_resources_belonging_to_them()
+    {
+        if (! config('hearth.resources.enabled')) {
+            return $this->markTestSkipped('Resource support is not enabled.');
+        }
+
+        $user = User::factory()->create();
+        $resource = Resource::factory()->create(['user_id' => $user->id]);
+
+        $resource->setTranslation('title', 'en', 'title in English');
+        $resource->setTranslation('title', 'fr', 'title in French');
+
+        $this->assertEquals($resource->getTranslation('title', 'en'), 'title in English');
+        $this->assertEquals($resource->getTranslation('title', 'fr'), 'title in French');
```

Comment:
Try also adding some a test asserting that `$resource->title` equals the English title, then set the application locale to `fr` and assert that `$resource->title` equals the French title. (Docs: https://laravel.com/docs/9.x/localization#configuring-the-locale)